elements/rh-subnav/rh-subnav.css

```diff
@@ -43,37 +44,41 @@
 }
 
 ::slotted(a) {
-  display: block;
-  white-space: nowrap;
-  padding: var(--rh-space-lg, 16px) var(--rh-space-2xl, 32px);
-  border-block-end: var(--rh-border-width-lg, 3px) solid transparent;
-  text-decoration: none;
-  color: var(--_subnav-link-text-color, #6a6e73);
+  display: block !important;
+  white-space: nowrap !important;
+  font-size: var(--rh-font-size-body-text-md, 1rem) !important;
+  padding: var(--rh-space-lg, 16px) var(--rh-space-2xl, 32px) !important;
+  border-block-end: var(--rh-border-width-lg, 3px) solid transparent !important;
+  text-decoration: none !important;
+  color: var(--_subnav-link-text-color, #6a6e73) !important;
 }
 
 ::slotted(a:hover),
 ::slotted(a:active),
 ::slotted(a[active]) {
-  color: var(--_subnav-link-hover-text-color, #151515);
+  color: var(--_subnav-link-hover-text-color, #151515) !important;
 }
 
 ::slotted(a:hover) {
-  border-block-end-color: var(--_subnav-link-hover-border-end-color, #d2d2d2);
+  border-block-end-color: var(--_subnav-link-hover-border-end-color, #d2d2d2) !important;
 }
 
 ::slotted(a:active),
 ::slotted(a[active]) {
-  border-block-end-color: var(--_subnav-link-active-border-end-color, #ee0000);
+  border-block-end-color: var(--_subnav-link-active-border-end-color, #ee0000) !important;
 }
 
 ::slotted(a:focus-visible) {
-  outline: var(--rh-border-width-md, 2px) solid  var(--_subnav-link-focus-outline-color, #0066cc);
-  outline-offset: -8px;
-  border-radius: 10px;
+  outline: 
+    var(--rh-border-width-md, 2px) 
+    solid 
+    var(--_subnav-link-focus-outline-color, #0066cc) !important;
+  outline-offset: -8px !important;
+  border-radius: 10px !important;
```

Comment:
@zeroedin the focus state does something weird:

<img width="309" alt="image" src="https://user-images.githubusercontent.com/1890950/223721386-a7bc6011-4ba1-4f66-87d4-6129969fea1b.png">

It seems to be the `border-radius: 10px !important;` on `::slotted(a:focus-visible)`, and I'm not sure if there's a good way to achieve a radius on the outline, but not the bottom border beyond increasing the complexity of the CSS. Might need to remove the outline on the `<a>` and apply it to a pseudo element. Or, perhaps it'd be simpler to apply the `border-bottom` to a pseudo element.